CMakeLists.txt

```diff
@@ -79,6 +79,13 @@ cmake_dependent_option(BUILD_SAMPLES "Build samples?" ON "BUILD_DISPATCHER" OFF
 cmake_dependent_option(BUILD_TOOLS "Build tools?" "${BUILD_ALL}" "BUILD_SAMPLES" OFF)
 option(BUILD_TESTS "Build tests?" "${BUILD_ALL}")
 
+set(IS_CMAKE3 OFF)
```

Comment:
Why do you need this? I see that both add_test() and enable_testing() present in cmake-2.8.5: https://cmake.org/cmake/help/v2.8.5/cmake.html. I think you can try cmake-2.8 on CentOS.... They have both packages cmake and cmake3...

Reply:
Indeed, the whole setup works with cmake 2.8.12.2 at least.

Reply:
ok, update, please

Reply:
CMake 3 is needed for compiling the latest Googletest on CentOS (that is needed for the UT). See https://github.com/Intel-Media-SDK/MediaSDK/wiki/Build-Media-SDK-on-CentOS (Packages needed for Googletest)

Reply:
But as of now we don't build googletests within building mediasdk. So, this check is really not needed. Google test framework could be either build or installed on some systems - that's responsibility of the user.